components/mail/thunderbird/Makefile

```diff
@@ -83,9 +90,12 @@ MOZCONFIG_DEBUG =	/dev/null
 MOZCONFIG_NON_DEBUG =	$(MOZCONFIG)
 endif
 
+COMPILER = gcc
+
```

Comment:
COMPILER = gcc is the default in shared-macros.mk and should not be necessary here.